shared_kv_storage/kv_storage/remote_lmdb/db_store_csv.i

```diff
@@ -0,0 +1,67 @@
+/* Copyright 2020 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+%module db_store_csv
+
+%include <std_vector.i>
+%include <std_map.i>
+%include <std_string.i>
+
+namespace std {
+    %template(StringVector) vector<string>;
+    %template(StringMap) map<string, string>;
+}
+
+%include <exception.i>
```

Comment:
Can we include `db_store.i` and then you can add additional include files specific to csv like 
```
%include <db_store.i>
%{
#include "db_store_csv.h"
%}

%include "db_store_csv.h"
```

Reply:
Learning for me. Thanks.